datacite-rest-client/src/test/java/org/gbif/datacite/util/IOUtils.java

```diff
@@ -0,0 +1,43 @@
+package org.gbif.datacite.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+/**
+ * Provides utility methods for IO related actions.
+ */
+public class IOUtils {
+
+    /**
+     * For getting file content by name as a string.
+     *
+     * @param name file name
+     * @return character data (String)
+     * @throws IOException in case of some input/output problems
+     */
+    public static String getResourceAsString(String name) throws IOException {
+        InputStream input = IOUtils.class.getClassLoader().getResourceAsStream(name);
```

Comment:
maybe something like this does the job new String (Files.readAllBytes(name File(IOUtils.class.getClassLoader().getResource(name).toURI())), StandardCharsets.UTF_8); ?